spec/models/order_item_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe OrderItem, type: :model do
+  let!(:order_items) {FactoryBot.create :order_item, unit_price: 150, quantity: 3}
+
+  describe "Associations" do
+    it {expect(order_items).to belong_to(:order)}
+    it {expect(order_items).to belong_to(:dish)}
+  end
+
+  describe "#unit_price" do
+    it "correct unit price" do
```

Comment:
gọi subject ở bên ngoài đi e
subject{order_items.send(:unit_price)}
như vậy k bị lặp code
bên trong các it thì là
is_expected.to eq

Reply:
Ok chị